Busybox tries several dhcpclients in a specific order, if one is found as
executable it gets the honor of asking for an address. By adding cirros as the
first option we make sure the /sbin/ifupdown script is used.

--- a/networking/ifupdown.c	2014-07-30 11:38:03.708210386 +0200
+++ b/networking/ifupdown.c	2014-07-30 11:38:24.106941433 +0200
@@ -521,6 +521,10 @@
 };
 
 static const struct dhcp_client_t ext_dhcp_clients[] = {
+	{ "cirros",
+		"/sbin/ifupdown up",
+		"/sbin/ifupdown down",
+	},
 	{ "dhcpcd",
 		"dhcpcd[[ -h %hostname%]][[ -i %vendor%]][[ -I %client%]][[ -l %leasetime%]] %iface%",
 		"dhcpcd -k %iface%",
